lib/options/dcu.robot

```diff
@@ -0,0 +1,29 @@
+*** Settings ***
+Documentation       Library for UEFI configuration using Dasharo Configuration
+...                 Utility tool. Commonly used when serial port is not
+...                 available.
+
+Library             Collections
+Library             OperatingSystem
+Library             String
+
+
+*** Keywords ***
+Set UEFI Option
+    [Documentation]    Set an UEFI option to a value.
+    [Arguments]    ${option_name}    ${value}
+    Run    git clone https://github.com/Dasharo/dcu
+    # TODO: Remove once smmstore support is merged
+    Run    cd dcu && git checkout smmstore > /dev/null 2>&1 && cd ..
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    Get Flashrom From Cloud
+    Execute Linux Command    flashrom -p internal -r coreboot.rom --fmap -i FMAP -i SMMSTORE &> /dev/null
```

Comment:
It could work, but if we want to make variables available for modification in OS, we can expose them to https://github.com/rhboot/efivar . I believe the only reason this doesn't work right now is because all Dasharo variables have `EFI_VARIABLE_BOOTSERVICE_ACCESS` attribute set

Reply:
Can we use `Execute Command In Terminal` here? We have multiple kwds with similar role, trying to get rid of it.

https://github.com/Dasharo/open-source-firmware-validation?tab=readme-ov-file#keywords

Reply:
changed in b039c87a3739164c8d73243438a7576bb0924fe0